htrace-zipkin/src/test/java/org/apache/htrace/zipkin/ScribeThriftSender.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.htrace.zipkin;
+
+import org.apache.htrace.Transport;
+import org.apache.htrace.core.*;
+import org.apache.htrace.impl.ScribeTransport;
+import org.apache.htrace.impl.ZipkinSpanReceiver;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import zipkin.collector.CollectorMetrics;
+import zipkin.collector.scribe.ScribeCollector;
+import zipkin.storage.InMemoryStorage;
+import zipkin.storage.QueryRequest;
+
+import java.util.List;
+
+public class ScribeThriftSender {
```

Comment:
conventionally, I think this should be named ScribeThriftSenderTest as sometimes things like this accidentally don't execute.